src/functions/avl-data-endpoint/index.ts

```diff
@@ -51,7 +51,7 @@ const processHeartbeatNotification = async (
 
     await putDynamoItem(tableName, subscription.PK, "SUBSCRIPTION", {
         ...subscription,
-        heartbeatLastReceivedDateTime: data.HeartbeatNotification.RequestTimestamp,
+        heartbeatLastReceivedDateTime: getDate(data.HeartbeatNotification.RequestTimestamp).toISOString(),
```

Comment:
It might be worth updating the schema for `RequestTimestamp` to validate it as a datetime: https://zod.dev/?id=datetimes so that this function will return a 400 zod error instead of a 500. What do you think?